projects/crud/src/lib/components/listing/listing.component.ts

```diff
@@ -40,7 +41,9 @@ export class ListingComponent implements OnInit, AfterViewInit {
     initialLoading = true;
     userHasPermission = true;
     pages: number;
+    gridData;
     @ViewChild('searchComponent', { read: ViewContainerRef, static: false }) searchComponent: ViewContainerRef;
+    @ViewChild('customComponent', { read: ViewContainerRef }) customComponent: QueryList<ViewContainerRef>;
```

Comment:
@lamiselzobeidy I don't think it returns type `QueryList`, might be `ViewContainerRef`